### ISSUE TITLE
rel attributes are reordered in 20220608.1

### ISSUE THREAD

**Issue by damianszczepanik** [OWASP/java-html-sanitizer#336]
Upgrading to version 20240325.1 (previous 20220608.1 works fine) makes my testing failing because order of REL values changes. What is worse when I fix the order it passes with JDK17 but fails with JDK11

Test that starts failing
https://github.com/damianszczepanik/cucumber-reporting/blob/8aed0cc6e6c41413b87047d0b89f73d750d37206/src/test/java/net/masterthought/cucumber/generators/EscapeHtmlReferenceTest.java#L66

Error message for JDK11
```
expected: "<a href="www.example.com" rel="nofollow noreferrer noopener">a hyper web reference</a>"
 but was: "<a href="www.example.com" rel="nofollow noopener noreferrer">a hyper web reference</a>"
```

PR with build/test output https://github.com/damianszczepanik/cucumber-reporting/pull/1179

Basically having this implementation is hardly possible to maintain code that is tested against different JVMs

**Comment by jmanico** [comment]
This was fixed by commit e48c0e3 ("Preserve order of rel attributes") and shipped in release 20260313.1. The sanitizer now keeps `rel` values in the order they were given while still de-duplicating them. Closing; please reopen if you still see reordering on 20260313.1 or later.
